last_task/server/server.go

```diff
@@ -0,0 +1,206 @@
+package main
+
+import (
+	"database/sql"
+	"encoding/json"
+	"fmt"
+	"log"
+	"net/http"
+	"time"
+
+	"github.com/pkg/errors"
+
+	_ "github.com/lib/pq"
+)
+
+type DB struct {
+	Conn *sql.DB
+}
+
+const (
+	id_key      = "3b2ea0ea927a17e540f2330bcafdaba3" //my id_key for api.openweathermap.org
+	serverPort  = 8080
+	dbType      = "postgres"
+	dbPort      = 5432
+	table       = "requests"
+	userDB      = "root"
+	pwdDB       = "root"
+	hostDB      = "database"
+	weatherHost = "https://api.openweathermap.org/data/2.5/weather?q=%s&appid=%s" //with get
+)
+
+type Request struct {
```

Comment:
Since you have one repo for client and server, better to define shareable structs in one file and import it to modules.